FhirToDataLake/NuGet.Config

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Just want to double confirm do we need to add this feed manually? Somehow I remember DevOps artifact is able to access public NuGet source.